docker-compose-services/postgres/docker-compose.postgres.yaml

```diff
@@ -19,6 +19,7 @@ services:
         source: "."
         target: "/mnt/ddev_config"
       - ddev-global-cache:/mnt/ddev-global-cache
+      - "./pgsql-db:/docker-entrypoint-initdb.d"
```

Comment:
Is there a missing file on the PR?

Reply:
Not sure what you mean. 

- `./pgsql-db` is a local folder the docs mention to create. 
- `/docker-entrypoint-initdb.d`  is an endpoint used by the image.

Are you getting an error?